apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -99,6 +99,80 @@ export const pageRouter = router({
         .execute()
     }),
 
+  getCategories: protectedProcedure
```

Comment:
to replace this nested inner join for better performance and more suitable for production use
https://github.com/opengovsg/isomer/pull/1056

Reply:
actually good call! benched this against local (10k rows) - took ~10s. lemme take a stab at refactoring this first before i use 1056!

Reply:
updated the query - ran against MOH newsroom, about ~300ms consistently! local (10k rows) took around 80ms - i think the difference is on the initial select, lemme see what i can do

<img width="1004" alt="image" src="https://github.com/user-attachments/assets/79bcf802-0077-4e28-a491-bb66da124677" />